src/mpyl/reporting/formatting/markdown.py

```diff
@@ -81,17 +82,20 @@ def markdown_for_stage(run_result: RunResult, stage: Stage):
         return ""
 
     result = f"{stage_to_icon(stage)} {__to_oneliner(step_results, plan)}  \n"
-    test_artifacts = _collect_test_artifacts(step_results)
-    test_results = _collect_test_results(test_artifacts)
+    test_artifacts: dict[str, JunitTestSpec] = _collect_test_artifacts(step_results)
+    test_results: dict[str, list[TestSuite]] = _collect_test_results(test_artifacts)
+
+    listoflists: list[list[TestSuite]] = [value for key, value in test_results.items()]
+    suites: list[TestSuite] = list(itertools.chain.from_iterable(listoflists))
```

Comment:
I would do this in one line

Reply:
Resolved in https://github.com/Vandebron/mpyl/pull/185/commits/383c76e60e9b31de361c83c6a9fa445a39474ab2